### PR TITLE
Added ProxyConfig to ntc_config_command and the ability to pass a string to ntc_show_command

### PR DESCRIPTION
**Pull request by sheffner1226**
Added ProxyCommand functionality to ntc_config_command to enable its use with Cisco CML VIRL (same as the ProxyCommand capability added to ntc_show_command in a previous release).

Added the ability to pass a string variable into ntc_show_command (in_string) for TextFSM parsing.  This will allow us to use standard Ansible modules to manage the connection to network devices while using ntc_show_command to parse the output without having to use temporary text files (the only currently supported method).

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
Something happened on config command, and it rewrote the entire file, I can't tell what is changed. Can you redo that? 

Can we also change it to raw_output to be consistent with raw_file?

File: library/ntc_config_command.py [hunk 1]
```diff
@@ -1,256 +1,288 @@
...
+    - netmiko
+options:
+    connection:
+        description:
+            - connect to device using netmiko or read from offline file
+              for testing
+        required: false
+        default: ssh
+        choices: ['ssh', 'telnet']
+    connection_args:
```
Comment (jmcgill298) [inline library/ntc_config_command.py]: This should follow the implementation in `ntc_show_command`

File: library/ntc_config_command.py [hunk 2]
```diff
@@ -1,256 +1,288 @@
...
+        description:
+            - Path to SSH configuration file for use by netmike, required when using proxy
```
Comment (jmcgill298) [inline library/ntc_config_command.py]: misspelled netmiko

File: library/ntc_config_command.py [hunk 3]
```diff
@@ -1,256 +1,288 @@
...
+            try:
+                if commands_file:
+                    if os.path.isfile(commands_file):
+                        with open(commands_file, 'r') as f:
+                            output = device.send_config_set(f.readlines())
+            except IOError:
+                module.fail_json(msg="Unable to locate: {}".format(commands_file))
+
+    if (error_params(platform, output)):
+        module.fail_json(msg="Error executing command:\
```
Comment (jmcgill298) [inline library/ntc_config_command.py]: It looks like you have a carriage return instead of using a newline

File: library/ntc_show_command.py [hunk 4]
```diff
@@ -84,6 +84,14 @@
               be rendered with the the TextFSM template
         required: false
         default: null
+    in_string:
```
Comment (jmcgill298) [inline library/ntc_show_command.py]: Please open a separate PR per feature change

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

File: library/ntc_show_command.py [hunk 5]
```diff
@@ -370,6 +379,9 @@ def main():
     platform = module.params['platform']
     device_type = platform.split('-')[0]
     raw_file = module.params['file']
+# SH - not 100% sure this will work properly
+    in_string = module.params['in_string']
```
Comment (jmcgill298) [inline library/ntc_show_command.py]: this should be named `raw_string`

**Comment by jeffkala** [comment]
Should be closed and retested after #268 is merged.

**Comment by jeffkala** [comment]
closing. Please retest now that #268 is deployed and 1.0.0 of this collection is in ansible-galaxy now.

**Pull request closed without merging**